### ISSUE TITLE
TypeError: Cannot read property 'value' of null

### ISSUE THREAD

**Issue by orteth01** [benmosher/eslint-plugin-import#1589]
Running eslint in our repo fails for version 2.19.0 with this error `TypeError: Cannot read property 'value' of null`.
Rolling back to 2.18.2 resolves the issue.

It looks like the problem is potentially related to https://github.com/benmosher/eslint-plugin-import/commit/99b3fbf8f25686aa86ba80661e5ba94d85e8f3d3.  `node.source` is null in `ExportNamedDeclaration`, which causes the failure. It fails for `export class Component extends React.Component { ... }`, `export function Component() { ... }`, and `export const Component = () => { ... }`

Please let me know if there's more information I can share to help diagnose this issue.

**Comment by orteth01** [comment]
we have the following import related rules in our eslint config
```
    'import/extensions': 1,

    'import/prefer-default-export': 0,

    'import/no-extraneous-dependencies': [
      'error',
      {
        devDependencies: true,
        optionalDependencies: false,
        peerDependencies: false,
      },
    ],
```

**Comment by ljharb** [comment]
Thanks, will take a look.

**Comment by ljharb** [comment]
Any chance you have example code this fails on?

**Comment by ljharb** [comment]
This appears to have been fixed with v2.19.1, can you upgrade and confirm?

Will reopen if not.

**Comment by orteth01** [comment]
confirming that fixed it. Thank you!
